Hypnotic-Herons/bangazon-api
# Ls customer

[Link to ticket](#1 ) 
 ## changes made
1. created customer model, serializer, and viewset
1. added urls for the api to the django app
1. added urls routes for customers

## steps to test
1. pull the branch and check out to it `git fetch --all` `git checkout lsCustomer`
1. makemigrations and migrate
1. run the server && go to http://127.0.0.1:8000/customers/
1. try to add a customer, put a customer, get a customer
1. you should be unable to delete a customer from the webapp or a normal json call
